src/UIKit/UIEnums.cs

```diff
@@ -2461,6 +2461,9 @@ public enum UIWindowSceneSessionRole {
 
 		[Field ("UIWindowSceneSessionRoleExternalDisplay")]
 		ExternalDisplay,
+
```

Comment:
missing `[iOS (13, 0)]`
that will fail tests on older OS

Reply:
Entire enum already has:
```
	[iOS (13,0), TV (13,0), NoWatch]
```

Reply:
I should have hit the up arrow :)